scripts/release.sh

```diff
@@ -55,6 +55,12 @@ safe_checkout_main() {
   fi
 }
 
+update_links() {
+  local version=$1
+  local versionTag="v${version}"
```

Comment:
in the docs the version dirs are major.minor, but github tags are major.minor.patch